packages/features/shell/Shell.tsx

```diff
@@ -215,7 +215,7 @@ const useBanners = () => {
 const Layout = (props: LayoutProps) => {
   const banners = useBanners();
   const pathname = usePathname();
-  const isFullPageWithoutSidebar = pathname.startsWith("/apps/routing-forms/reporting/");
+  const isFullPageWithoutSidebar = pathname?.startsWith("/apps/routing-forms/reporting/");
```

Comment:
This is not part of this pr, but this type error is the reason currently user faced type error during build